herbert_core/utilities/general/is_folder.m

```diff
@@ -0,0 +1,26 @@
+function ok = is_folder(name)
+% tests if name is a valid folder
+%
+% Input:
+% ------
+%   name                The name of the file you want to check;
+%
+% Usage:
+% ------
+%  >> is_file('/home/user/test');        % True if folder exists
+%  >> is_file('test');                   % True if folder exists in current dir
+%
+
+    if ~verLessThan('matlab', '9.1') % R2016b
+        ok = isfolder(name);
+    else
+        [path,~,~] = fileparts(strtrim(name));
```

Comment:
this isn't how the code it replaces was implemented: `exist(target,'dir') == 7`

Reply:
True, but it reproduces what `isfolder` does.